jans-linux-setup/jans_setup/setup_app/installers/shibboleth.py

```
@@ -67,26 +67,26 @@ def create_directories(self):
 
     def create_idp_client(self):
         self.logIt("Creating Shibboleth IDP OAuth client")
-        
+
         _, jans_auth_config = self.dbUtils.get_jans_auth_conf_dynamic()
         Config.templateRenderingDict['jans_auth_token_endpoint'] = jans_auth_config['tokenEndpoint']
 
         jans_scopes = self.dbUtils.get_scopes()
```

Ruff (0.14.14)

[error] Local variable `jans_scopes` is assigned to but never used

Remove assignment to unused variable `jans_scopes`

(F841)